backup_and_restore/application_backup_and_restore/troubleshooting/oadp-pod-crash-resource-request-doc.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="oadp-pod-crash-resource-request-doc"]
+= Pods crashing or restarting due to lack of memory or CPU
+include::_attributes/common-attributes.adoc[]
+:context: oadp-troubleshooting
+
+
+toc::[]
+
+If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.
+
```

Comment:
Here we have:
```
You can use the configuration.velero.podConfig.resourceAllocations specification field in the oadp_v1alpha1_dpa.yaml file to set specific resource requests for a Velero pod.
```
Change oadp_v1alpha1_dpa.yaml to something more straight: 
Change DataProtectionApplication CR yaml

Reply:
I still see the phrase 'oadp_v1alpha1_dpa.yaml' in the preview.

Reply:
Not changing, it is fine as is

Reply:
I will look at it again when re-writing